cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -625,6 +625,15 @@
     </description>
   </property>
 
+  <property>
+    <name>plugin.exclude.with.requirements</name>
+    <value></value>
+    <description>
+      Comma separated list of requirements which cannot be meet in the instance and any plugins requiring either one
```

Comment:
Comma separated list of plugin requirements that cannot be met in the instance. Plugins can require any of these will be treated as if they don't exist. For example, if 'transactions' is given, any plugin that requires transactions will be treated as if they don't exist.

Reply:
this change requires a change to the hash in the docs file.

Reply:
Done.